lib/git_hooks/pre_commit/rspec.rb

```diff
@@ -18,6 +18,10 @@ def validate
 
         abort 'Prevented broken commit' if rspec_executor.errors?
       end
+
+      protected
```

Comment:
I have the tendency to hide the readers. This kind of helps me avoid violating Demeter's law.